pkg/storage/storage.go

```diff
@@ -34,28 +32,28 @@ import (
 // nice if the kubelet told us this in the summary API...
 var kubernetesCadvisorWindow = 30 * time.Second
 
-// sinkMetricsProvider is a provider.MetricsProvider that also acts as a sink.MetricSink
-type sinkMetricsProvider struct {
+// Storage is a provider.MetricsProvider that also acts as a sink.MetricSink
```

Comment:
1) s/provider.MetricsProvider/api.MetricsProvider
2) There is no MetricSink anymore